adserver/templates/adserver/includes/flight-metadata.html

```diff
@@ -0,0 +1,73 @@
+{% load i18n %}
+
+
+{# Used in flight reporting and flight management #}
+
+
+<dl>
+  {% if flight.campaign.campaign_type == 'community' %}
+    <dt>{% trans 'Community ad' %}</dt>
+    <dd>{% trans 'This ad is done for the benefit of the community' %}</dd>
+  {% endif %}
+  {% if flight.campaign.campaign_type == 'house' %}
+    <dt>{% trans 'House ad' %}</dt>
+    <dd>{% trans 'This is a house advertisement' %}</dd>
+  {% endif %}
+  {% if flight.campaign.campaign_type == 'paid' and flight.live %}
+    <dt>{% trans 'Value remaining' %}</dt>
+    <dd>${{ flight.value_remaining|floatformat:2 }} / ${{ flight.projected_total_value|floatformat:2 }}</dd>
+  {% endif %}
+  {% if flight.cpc %}
+    <dt>{% trans 'Cost per click (CPC)' %}</dt>
+    <dd>${{ flight.cpc|floatformat:2 }}</dd>
+  {% endif %}
+  {% if flight.cpm %}
+    <dt>{% trans 'Cost per 1,000 impressions (CPM)' %}</dt>
+    <dd>${{ flight.cpm|floatformat:2 }}</dd>
+  {% endif %}
+  {% if flight.start_date %}
+    <dt>{% trans 'Estimated start date' %}</dt>
+    <dd>{{ flight.start_date }}</dd>
+  {% endif %}
+  {% if flight.end_date %}
+    <dt>{% trans 'Estimated end date' %}</dt>
+    <dd>{{ flight.end_date }}</dd>
+  {% endif %}
+  {% with flight.campaign.publishers.count as publishers_count %}
+    {% if publishers_count %}
+      <dt>{% trans 'Publishers' %}</dt>
+      <dd>
+        <ul>
+          {% for publisher in flight.campaign.publishers.all|slice:":5" %}
```

Comment:
Doesn't seem like this is sorted by pageview or weight at all? This seems like a pretty bad thing to expose, since I don't think it will show what advertisers expect all the time, and could cause some confusion. I definitely think it should at least be weighted by impression count.

Reply:
I definitely think showing the top by weighted impressions is ideal. The question then becomes over what timeframe? I also would like to show this data even for upcoming flights so that advertisers know where their ad will be shown. I think both are useful data. However, I don't want to mislead. Maybe this requires more thought.

Reply:
> The question then becomes over what timeframe?

The timeframe of the report date filter? 

Upcoming seems a bit trickier, but that feels like a different thing than "where are my ads currently being shown" that is more like "Where are all the possible places my ad might be shown". Both are useful, but I think the first is the most important currently.

Reply:
> The timeframe of the report date filter?

This same data is shown on the flight detail view which doesn't have any dates associated with it. I think this data will be available exclusively on a per-publisher detail report.